spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/json/MappingAwarePageableArgumentResolver.java

```diff
@@ -66,7 +66,7 @@ public Pageable resolveArgument(MethodParameter parameter, ModelAndViewContainer
 		Pageable pageable = delegate.resolveArgument(parameter, mavContainer, webRequest, binderFactory);
 
 		if (pageable == null || pageable.getSort() == null) {
-			return null;
+			return pageable;
```

Comment:
Can we have a unit test that checks the outcome depending on what the delegate returns?